gcloud-java-compute/pom.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0"?>
```

Comment:
Are you sure about this? It seems to me it's using spaces (also I just copy pasted it from an existing pom of ours).